### ISSUE TITLE
DynamicPageTitle: A11y: Count of number of items in overflow popover not read by screen readers

### ISSUE THREAD

**Issue by abhijeet181088** [SAP/ui5-webcomponents-react#5926]
### Describe the bug

If there are more number of buttons in the DynamicPageTitle -> actions, extra buttons are pushed to the menu popover.
On accessing this overflow button and opening the popover, the count of items in the popover is not read by JAWS and NVDA screen readers in Windows system.
This works fine with macOS in-built screen reader.

### Isolated Example

https://stackblitz.com/edit/github-lg4jzd-ks2x4w?file=src%2FApp.tsx

### Reproduction steps

1. Go to the stackblitz example provided
2. Turn on JAWS/NVDA screen reader
3. Navigate to Overflow menu button in the DynamicPageTitle
4. Activate the button to open the popover 
5. JAWS/NVDA does not read the count of items in the popover (screenshot attached)
6. MacOS screeen-reader reads the count (screenshot attached)

### Expected Behaviour

JAWS/NVDA should read the count of items in the overflow popover

### Screenshots or Videos

<img width="965" alt="Windows+Jaws_CountNotRead" src="https://github.com/SAP/ui5-webcomponents-react/assets/26282517/0fc87ad9-ca41-454c-942b-2f46db6bd107">
<img width="1792" alt="Mac_CountIsRead" src="https://github.com/SAP/ui5-webcomponents-react/assets/26282517/a3d19b50-6ba2-4a34-9a52-9f577aabc1a5">

### UI5 Web Components for React Version

1.28.0

### UI5 Web Components Version

1.24.0

### Browser

Edge

### Operating System

Windows

### Organization

SAP

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @abhijeet181088

the Toolbar issue has been fixed in [v1.29.13](https://github.com/SAP/ui5-webcomponents-react/releases/tag/v1.29.13) and [v2.3.2](https://github.com/SAP/ui5-webcomponents-react/releases/tag/v2.3.2). In v1, the Toolbar is used internally by the `DynamicPageTitle`, so the fix is applied there as well.

To properly enable announcements, make sure to set the `data-accessible-name` prop on all components that implement `accessibleName` instead of `aria-label`. You can find more details here: https://sap.github.io/ui5-webcomponents-react/v1/?path=/docs/layouts-floorplans-toolbar--docs#announce-number-of-items-in-overflow-popover
